app/models/katello/content_view.rb

```diff
@@ -637,6 +638,14 @@ def check_ready_to_publish!(importing: false, syncable: false)
       true
     end
 
+    def check_repositories_blocking_publish!
+      repositories_with_blocking_tasks = repositories.select { |repo| repo.blocking_tasks }
+
+      if repositories_with_blocking_tasks.any?
```

Comment:
Not saying you have to use it, but `#any?` can take a block to save some allocations

Reply:
Using `any?` more as a conditional check to see if there are any entries in the task array rather than an enumerable to act on..